pages/metrics/metrics_per-org.html

```diff
@@ -59,10 +59,17 @@ <h4>{{block.meta.title}}</h4>
             <td><a href="{{column}}">{{column}}</a></td>
             {%- endif -%}
 
+            {%- else -%}
+            {%- assign pageURL = 5 -%}
+            {%- if block.data[0][forloop.index0] == "pageTitle" -%}
+              <td align="right">
+              <a href="{{row[pageURL]}}" target="_blank">{{column | toLocaleString | replace: " - Catalog", ""}}</a>
```

Comment:
what's this line doing exactly?

Reply:
if the current column in row corresponds to the `pageTitle` column then add the original catalog page ( `row[pageURL]` ) the file was clicked on as a hyperlink with some string munging. i created the `pageURL` var to stand out and describe what the index represents rather than just doing row[5].  I can try to change this to make it more readable. frankly i'm not super familiar with templating languages.

Reply:
i derive the dataset title from the html title using the `pageTitle` GA dimension which includes " - Catalog" at the end

Reply:
>i derive the dataset title from the html title using the pageTitle GA dimension which includes " - Catalog" at the end

the string replace was the part that got me. i guess we do it for every dataset so i expect it's fine.